site/pages/api/admin/delete-user.api.ts

```diff
@@ -27,14 +26,6 @@ const deleteUser = async (req: DeleteUserApiRequest, res: NextApiResponse): Prom
             throw Error("Insufficient values provided to delete a user");
         }
 
-        const userDetails = await getUserDetails(username);
-
-        const formattedUserDetails = user.parse(userDetails);
-
-        if (formattedUserDetails.organisation !== session.orgId) {
```

Comment:
We should probably keep this check in and only disable it in the case where the user is a sysadmin:

```ts
if (!session.isSystemAdmin && formattedUserDetails.organisation !== session.orgId)
```

Reply:
done